libs/dh/shared/navigation/src/breadcrumb.service.ts

```diff
@@ -0,0 +1,55 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { effect, inject, Injectable, signal } from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import { EventType, Router } from '@angular/router';
+import { filter } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
```

Comment:
nitpick: since the service is provided in `root`, it will "live" as long as the app is running (after being instantiated the first time). So the `navigationEnded` Signal, and as a consequence the `effect`, will run on every navigation. Not a big issue since the `clearBreadcrumbs` method is super simple but something to be aware of.

Reply:
Need it in root, to be able to access the values in the shell component